corehq/apps/api/domain_metadata.py

```diff
@@ -62,7 +76,16 @@ def obj_get_list(self, bundle, **kwargs):
         if kwargs.get('domain'):
             return [self.obj_get(bundle, **kwargs)]
         else:
-            return list(Domain.get_all())
+            filters = {}
+            if hasattr(bundle.request, 'GET'):
+                filters = bundle.request.GET
+            domains = list(Domain.get_all())
```

Comment:
In the meantime I've temporarily revoked `niketsoral@gmail.com`'s permission to access that API to remove burden from our servers.